build.xml

```diff
@@ -178,15 +178,15 @@
 		<if>
 			<istrue value="${ant.version.atleast.1.9.2}"/>
 			<then>
-				<javadoc sourcepath="${basedir}/src" access="public" postprocessgeneratedjavadocs="false" source="6" packagenames="*">
+				<javadoc sourcepath="${basedir}/src" access="public" postprocessgeneratedjavadocs="false" source="8" packagenames="*">
```

Comment:
I believe for 6.1, the source has to be "7", not "8". You cannot use Java "8" specific features until 7.x. See the help here - https://help.pentaho.com/Documentation/6.0/0D0/160/000 - they *have* to be able to run using Java 7.